io.github.prateekmedia.appimagepool.yml

```diff
@@ -0,0 +1,60 @@
+app-id: io.github.prateekmedia.appimagepool
+runtime: org.gnome.Platform
+runtime-version: "40"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+rename-icon: appimagepool
+rename-appdata-file: appimagepool.appdata.xml
+rename-desktop-file: appimagepool.desktop
+command: appimagepool
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --persist=Applications
+  - --talk-name=org.freedesktop.Notifications
+build-option:
+  - append-path: /usr/lib/sdk/rust-stable/bin
+    build-args:
+      - --share=network
+    env:
+      CARGO_HOME: /run/build/rust-flatpak/cargo
+modules:
+  - name: flutter
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://storage.googleapis.com/flutter_infra_release/releases/beta/linux/flutter_linux_2.6.0-5.2.pre-beta.tar.xz
+        sha256: 1a2d82b9a7934d00d680b6a0a9e09697264dd86ba37d984812c73297f33458e2
+    build-commands:
+      - install -d /app/lib/sdk/flutter
+      - cp -rpv * /app/lib/sdk/flutter/
+
+  - name: appimagepool
+    buildsystem: simple
+    build-commands:
+      - export FLUTTER_SDK=/app/lib/sdk/flutter
+      - export PATH=$PATH:$FLUTTER_SDK/bin
```

Comment:
You should just had a prepend-path: /app/lib/sdk/flutter/bin in the build-options (or add it to the `append-path`)